src/js/app.js

```diff
@@ -25,9 +25,10 @@ const app = new Vue({
   },
   template: `
     <div class="app">
-      <div class="search">
+    <section class="search">
+      <div class="search_bar">
           <div class="search_sort" v-bind:class="{ open: is_dropdown_open }">
-            <div class="search_sortTrigger" v-on:click="toggleDropdown"><span>{{ sort_options[sort] }}</span></div>
+            <div class="search_sortTrigger" v-on:click="toggleDropdown"><span><span class="search_sortIcon">⇵</span>{{ sort_options[sort] }}</span></div>
```

Comment:
Hmm... I wonder if it'd be better to use the escape code or not... It's probably fine either way, but the escape code would probably be more universally compatible.